### ISSUE TITLE
Update documentation for this repo

### ISSUE THREAD

**Issue by danmosemsft** [microsoft/msbuild#3928]
### Steps to reproduce

1. Clone repo on Windows
2. At root, type `build -skiptests`

### Result
```
c:\git\msbuild>build -skiptests
C:\git\msbuild\build\build.ps1 : Cannot bind parameter because parameter 'skiptests' is specified more than once. To provide multiple values to parameters that can accept multiple values, use the array syntax. For example, "-parameter
value1,value2,value3".
```

This is because `build.cmd` contains `powershell -NoLogo -NoProfile -ExecutionPolicy ByPass "%~dp0build\build.ps1" -build -skiptests %*`

However `build -?` shows the flag as valid
```
c:\git\msbuild>build -?
build.ps1 [-build] [-ci] [-configuration <string>] [-help] [-nolog] [-pack] [-prepareMachine] [-rebuild] [-norestore] [-sign] [-skiptests] [-test] [-bootstrap] [-verbosity <string>] [-hostType <string>] [-DotNetBuildFromSource] [-DotNetCoreSdkDir <string>] [-properties <string[]>] [<CommonParameters>]
```

I am not sure what is expected

**Comment by danmosemsft** [comment]
https://github.com/dotnet/source-indexer/pull/15

**Comment by rainersigwald** [comment]
I expect this to churn again when we move to Arcade, but we should indeed have matching docs and behavior.

Right now, `build.cmd` builds but does not test (enable with `-test`) or rebuild with the just-built MSBuild (enable with `-bootstrap`).

**Comment by livarcocc** [comment]
Yeah, arcade will change this again, but by then, we will all be in one single infra.

**Comment by livarcocc** [comment]
@BenVillalobos since you were the last one to ramp up on this repo, would you like to take a stab at improving our docs here? Does not to be perfect on the first turn of the crank, but better and more accurate with how things are done now.

**Comment by BenVillalobos** [comment]
Fixed with #4280
